src/packaging/docker-services/generate-ssl-stores/docker-entrypoint.sh

```diff
@@ -73,24 +80,33 @@ then
         esac
     done
 
-fi
+ fi
 set -x
 
 for store_name in ${CASSANDRA_KEYSTORE} ${REAPER_KEYSTORE} ${GENERIC_TRUSTSTORE}
 do
     rm -f ${store_name}
 done
 
+if [ -f "$CA_KEY" ] ; then
+    echo "$CA_KEY" exists, will not create
+else
+   openssl genrsa -out ${CA_KEY} 2048
+fi
+
+if [ -f "$ROOT_CA_CERT" ] ; then
+    echo "$ROOT_CA_CERT" exists, will not create
+else
+# Create the Root Certificate Authority (Root CA) from the Certificate Authority Configuration and verify contents.
+   openssl req -x509 -sha256 -new -nodes -key ca-key -out ${ROOT_CA_CERT} -config ${CA_CERT_CONFIG}
+fi
```

Comment:
Similar to my previous comment about the container being an example, remove the checks to see if the `ca-key` exists. Originally the `ca-key` was generated further down as part of the Root CA creation. Hence, there was never any need to check for it existing.

I do like the idea of showing the ca-key generation:
```
openssl genrsa -out ca-key 2048
```

I also like the Root CA generation command:
```
openssl req -x509 -sha256 -new -nodes -key ca-key -out ${ROOT_CA_CERT} -config ${CA_CERT_CONFIG}
```

I think it would be better to add these lines after we `echo "Generic Certificate Authority configuration"`. Maybe replace [line 92](https://github.com/thelastpickle/cassandra-reaper/pull/951/files#diff-ac79225b81efb4bc9081fc4c103f1cd7L92) with the above two lines. So the final blow should look like this:

```
# Create Certificate Authority Key, then create the Root Certificate Authority (Root CA) from the Certificate Authority Configuration and verify contents.
openssl genrsa -out ca-key 2048
openssl req -x509 -sha256 -new -nodes -key ca-key -out ${ROOT_CA_CERT} -config ${CA_CERT_CONFIG}
openssl x509 -in ${ROOT_CA_CERT} -text -noout
```